docs/ExposePortsToInternet.md

```diff
@@ -0,0 +1,46 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Expose Container Port to Host-Interface / Internet
+
+In the [Calico without Docker networking tutorial](calico-with-docker/without-docker-networking/README.md) 
+or the [Calico as a Docker network plugin tutorial](calico-with-docker/docker-network-plugin/README.md)
+we created containers and assigned endpoints (a container interface) to them. This is used for Container-
+To-Container communication.
+
+The example below shows how to expose a port of a container to the Host-Interface so this container is 
+reachable from outside / the internet.
+
+## Why isn't the `-p` flag on `docker run` working as expected?
```

Comment:
This sounds like there is a problem with using Calico.

Can you have this as basic text, rather than a heading, and include it in the paragraph below as something like:
"Calico creates a new interface on the container to enforce security rules when networking, so calling  `docker run` with the `-p` flag will not actually apply the port-forwarding to the Calico interface..."